scripts/sysman/device.yml

```diff
@@ -100,14 +117,40 @@ members:
       desc: "[out] Indicates if the device has been repaired"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Device reset properties"
+class: $sDevice
+name: $s_reset_properties_t
+base: $s_base_properties_t
+members:
+    - type: ze_bool_t
+      name: force
+      desc: |
+            [in] If set to true, all applications that are currently using the device will be forcibly killed.
+    - type: $s_reset_type_t
+      name: reset_type
```

Comment:
struct fields are typically named using camelCase style. Recommend renaming `reset_type` to `resetType`.

Reply:
Fixed